samples/apps/copilot-chat-app/scripts/README.md

```diff
@@ -0,0 +1,48 @@
+# Copilot Chat Setup Scripts
+
+> The PowerShell scripts in this directory require [PowerShell Core 6 or higher](https://github.com/PowerShell/PowerShell#get-powershell).
+
+## Before You Begin
+To run these scripts, you will need the following:
+- *YOUR_CLIENT_ID*
+  - This is the client ID (also known as application ID) associated with your Azure Active Directory (AAD) application registration, which you can find in the Azure portal.
+- *YOUR_TENANT_ID*
+  - This is the tenant ID (also known as directory ID) associated with your AAD app registration, which you can find in the Azure portal. If you are using a personal MSA account, you can simply enter `msa` for this value to use the personal MSA tenant.
+- *YOUR_API_KEY*
```

Comment:
Can we rename this MY_AZUREOPENAI_OR_OPENAI_KEY to keep naming terminology consistent with the main copilot chat readme. We also use a lot of API keys at this point and being specific will create clarity.